Update wi-fi-settings-android-enterprise.md (#3599)
MicrosoftDocs/memdocs: memdocs/intune/configuration/wi-fi-settings-android-enterprise.md

  - **Automatic**: Use a file to configure the proxy server. Enter the **Proxy server URL** that contains the configuration file. For example, enter `http://proxy.contoso.com`, `10.0.0.11`, or `http://proxy.contoso.com/proxy.pac`.

    For more information on PAC files, see [Proxy Auto-Configuration (PAC) file](https://developer.mozilla.org/docs/Web/HTTP/Proxy_servers_and_tunneling/Proxy_Auto-Configuration_(PAC)_file) (opens a non-Microsoft site).
    
    > [!NOTE]

>     > When a device is marked corporate, the user gets to have little control and all control will be with the policies enforced.
>     > So when  Wi-Fi Profile is pushed for the device, by design Wi-Fi will be made to turn on.

Reviewer: ```suggestion
    > When a device is marked **Corporate**, the user has little control, and all control will depend on the policies enforced. So when a Wi-Fi Profile is pushed for the device, Wi-Fi will be turned on by design.
```

Reviewer 2: This is my suggested change:

> When a device is marked as corporate during enrollment (organization-owned), policies control device features and settings. Users can be prevented from managing features and settings in the policy.
>
> When a Wi-Fi policy is assigned to devices, then Wi-Fi is enabled, and users can be prevented from turning off Wi-Fi.
